src/docs/ftw-introduction/ftw-product/index.md

```diff
@@ -0,0 +1,132 @@
+  example:
+  - _EditListinWizard_ is moved under _EditListinPage_
+  - Search map and filters are moved under _SearchPage_
+  - _TransactionPage_ got subcomponents like _ActivityFeed_,
+    _ReviewModal_, _ReviewForm_, _SendMessageForm_, and
+    _TransactionPanel_.
+- **src/forms**: this directory was removed as there was only one form
+  that was truly shared between pages: _StripeConnectAccountForm_. It's
+  moved to _src/components/_.
+
+TODO: add other notable changes here
+
+## Differences in transaction process
+
+FTW-product has different transaction process graph than what FTW-daily
+and FTW-hourly use. Visually, the biggest changes are after checkout
+page and before review period. E.g. FTW-product has dispute state there.
+However, the defining difference is actually in actions: FTW-product
+doesn't use bookings (and availability management of time-slots).
+Instead, it has actions to deal with stock management.
+
+Roughly, the transaction process graph looks like this:
+
+![flex-product-default-process](./flex-product-default-process.png)
+
+## Managing stock
+
+Stock management is pretty straightforward. On EditListingWizard, (which
+is a component to create and update listings), provider can set the
+stock quantity for the listing. Under the hood, FTW-product calls API
+with _`sdk.stockAdjustments/compareAndSet()`_ with the value provider is
+trying to set (`newTotal`) and also the previous value (`oldTotal`). At
+the time of update, if the assumed old total still matches with the
+current stock, the **currentStock** entity is updated to the value
+passed through with `newTotal` parameter. This check is done to prevent
+race conditions, where someone orders a listing while provider is
+updating its stock.
+
+Listing entity has a relationship to "currentStock" entity - so,
+currentStock info can be included listing queries. Then on ListingPage,
+customer can order as many listings as there is stock left for it.
+
+> Note: Flex API and FTW-product don't yet support listing variants (the
+> stock of yellow t-shirts vs red t-shirts). So, listing variants need
+> to be modelled with separate listings.
+
+## Shipping the product
+
+For product marketplaces, it's usually necessary to ask shipping address
+from customer. FTW-product ask shipping method on ListingPage and if
+'shipping' is chosen, shipping address is asked on checkout page. The
+address is saved to transaction's protected data in transitions:
+_request-payment_ and _request-payment-after-enquiry_.
+
+On TransactionPage (the page opened by clicking inbox items), shipping
+address or pickup address is shown to both parties of the transaction.
+Provider has 2 weeks to mark the item shipped
+(_transition/mark-delivered_) or the automatic expiration happens
+(_transition/auto-cancel_) and customer is refunded. If needed, customer
+can dispute the order from "Delivered" state.
+
+![Dispute modal](./dispute-modal.png)
```

Comment:
This was the first mention of Sneakertime. Should be briefly introduce the concept in the beginning of the article? Something like:

> While FTW daily was about renting saunas and FTW hourly was about booking Yoga teachers, FTW product is a conceptual marketplace called Sneakertime to buy sneakers.